tests/test_update_metrics.py

```diff
@@ -0,0 +1,41 @@
+import csv
+import os
+import shutil
+import tempfile
+import unittest
+import src.tasks.tasks as tasks
+import torch
+
+
+class TestUpdateMetricsAccuracy(unittest.TestCase):
+    def setUp(self):
+        self.temp_dir = tempfile.mkdtemp()
+        self.path = os.path.join(self.temp_dir, "temp_dataset.tsv")
+        self.task = tasks.QQPTask("", 100, "qqp", tokenizer_name="MosesTokenizer")
+        predictions = torch.Tensor([[[1, 0], [1, 0]], [[1, 0], [0, 1]]])
```

Comment:
Explain the shape of this tensor. It's impossible to follow the test otherwise, and tests really need to be self-explanatory.